server/render.js

```diff
@@ -31,3 +31,39 @@ exports.renderReduxComponent = function renderReduxComponent(
     )
   );
 };
+
+/**
+ * This namespace a helper method for SSR using @loadable/component
+ * ```javascript
+ * import { ChunkExtractor } from "@loadable/server";
+ * import {renderReduxComponent} from "@quintype/framework/server/render";
+ * import path from "path";
+ *
+ * const statsFile = path.resolve("stats.json"); // This is the stats file generated by webpack loadable plugin
+ * const extractor = new ChunkExtractor({ statsFile, entrypoints: ["topbarCriticalCss", "navbarCriticalCss"] }); // We create an extractor from the statsFile
```

Comment:
I think the chunk extractor needs to go inside the function.

Reply:
It should be outside for the benefit of critical css